Example/EditorDemo/WPViewController.m

```diff
@@ -123,6 +123,31 @@ - (void)editorViewController:(WPEditorViewController *)editorViewController imag
     [self.mediaAdded removeObjectForKey:imageId];
 }
 
+- (void)editorViewController:(WPEditorViewController *)editorViewController imagePasted:(UIImage *)image
+{
+    NSString *imageID = [[NSUUID UUID] UUIDString];
+    NSString *path = [NSString stringWithFormat:@"%@/%@.jpg", NSTemporaryDirectory(), imageID];
+    NSData *imageData = UIImagePNGRepresentation(image);
+    [imageData writeToFile:path atomically:YES];
+
+    [self.editorView insertLocalImage:[[NSURL fileURLWithPath:path] absoluteString] uniqueId:imageID];
+
+    NSProgress *progress = [[NSProgress alloc] initWithParent:nil userInfo:@{ @"imageID": imageID, @"url": path }];
```

Comment:
Starting in this line, the code seems very similar to [this](https://github.com/wordpress-mobile/WordPress-Editor-iOS/blob/develop/Example/EditorDemo/WPViewController.m#L313).

I'd suggest adding a method to handle it instead.